templates/pricing/desktop.html

```diff
@@ -10,60 +10,57 @@
   https://docs.google.com/document/d/1xVtaZ7wr5sdBqye3Gc-XWXEpNqXaol8uDdMwJ_KyOO8/edit
 {% endblock meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
-  <section class="p-strip--suru-topped">
-    <div class="row">
-      <div class="col-8">
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
         <h1>
           Ubuntu Desktop
-          <br />
+          <br class="u-hide--small" />
           Enterprise Services
         </h1>
+      </div>
+      <div class="col">
         <p>Day zero support for Ubuntu Desktop deployment, compliance and custom configuration.</p>
-        <a class="p-button"
-           href="https://assets.ubuntu.com/v1/a48fc9cd-Ubuntu%20Desktop%20DS%2021.06.23.pdf">Download the datasheet</a>
-        <a href="/desktop/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+        <div class="p-cta-block">
```

Comment:
The `p-cta-block` is introducing additional padding compared to the Design
![Screenshot 2024-10-02 at 11 00 25 AM](https://github.com/user-attachments/assets/92f4f9cd-aa5f-41e9-ab93-c7aa61f779c6)

<img width="395" alt="Screenshot 2024-10-02 at 11 00 51 AM" src="https://github.com/user-attachments/assets/c8c1ca66-eb1f-477d-b9ab-a4f11f3ac5e0">

Reply:
@mattea-turic What do you think here? As I understand, we should be standardising all our CTA blocks and that's why the class exists, which would make this more of a Vanilla issue. But I am happy to update if needed